packages/lesswrong/themes/siteThemes/eaTheme.ts

```diff
@@ -138,6 +138,7 @@ export const eaForumTheme: SiteThemeSpecification = {
         },
         postStyle: {
           ...basicText,
+          fontFamily: sansSerifStack,
```

Comment:
This seems too risky to change, plus I think "postStyle" should in fact be our serif font, because that's the primary font family used in posts. Could you change the fonts in the specific places they are used? You can do so without having to forum-gate any of the changes, because we are the only site that uses these specific search results components.

Also the font used for the post snippet text should be our serif font, because it's text from the body of the post:
<img width="541" alt="Screen Shot 2023-04-07 at 2 00 51 PM" src="https://user-images.githubusercontent.com/9057804/230655840-8e0ed4bd-b1b2-4f25-9b62-cfa4bb049cc8.png">

Reply:
Oh, whoops - I missed that this is the value used for posts (that's what I get for doing bugfixes late at night). Should be fixed now!